aiohttp/web_fileresponse.py

```diff
@@ -222,12 +235,16 @@ def __init__(self, path: Union[str, pathlib.Path],
             return await super().prepare(request)
 
         if hdrs.CONTENT_TYPE not in self.headers:
-            ct, encoding = mimetypes.guess_type(str(filepath))
+            # Guess Content-Type and encoding of original path,
```

Comment:
I think you can register new mime types there.

Reply:
I thought about that, but there is no consensus about the filename/encoding name (see https://bugs.python.org/issue32021). Also, registering mimetypes in request handlers which might be seldom/never called seemed wrong, with a good posibility of creating hard to track down bugs for the user of the library.